### DISCUSSION TITLE
Podman desktop compatibility with podman

### DISCUSSION THREAD

**Discussion by alex-martel** [podman-desktop/podman-desktop#6496] [General]
I would like to know if there's a sort of compatibility matrix of podman-desktop and podman. I am asking because of this issue I raised here; https://github.com/containers/podman/issues/22121 where I updated podman to 1.8.0, running podman 4.9.3, and it worked all good, then my podman updated from 4.9.3 to 5.0.0 via brew but then I realized it is actually not compatible for osx monteray (which I cannot update due to having an older macbook).

I had to downgrade podman to 4.9.3 to get back to a working environment, but not I wonder if I keep updating podman-desktop, will it become eventually not compatible with podman 4.9.X.

**Comment by jeffmaury** [comment]
Podman 5.0 support in Podman Desktop in planned as experimental in 1.9.0 and full support in 1.10.0

**Comment by benoitf** [comment]
I think @manofthepeace want to know if Podman Desktop v1.10 will still support Podman v4 engine.

We're using as much as possible the compat/Docker API that is not evolving much and the same for Podman API (like for pods, etc) so it should work and for example we're already hiding/showing properties depending on the version of Podman (for example some stuff was not possible in podman v4.0 but was possible in podman v4.9)

All the basic stuff like listing/creating/deleting images, containers should continue to work for a while

**Reply by afbjorklund** [reply]
How long will you continue to support Podman v3, the default version provided for Ubuntu LTS and Debian?

* https://packages.ubuntu.com/podman

* https://packages.debian.org/podman

It is already too old to support Podman Machine, but it "limps along" for the basic of Podman Engine support

**Reply by benoitf** [reply]
I think it will continue to work as well for all the basic stuff for Podman v3

the Rest API is backward compliant so I think it can still work there. But Podman v3 is only for Linux (as for mac/Windows the local socket was only added with podman v4)

**Reply by afbjorklund** [reply]
The newer releases should have Podman 4.3.1, maybe that will continue to work for a while (latest has 4.9.3)

Right, the older releases only supported `ssh:` connections - as they didn't have the `unix:` tunneling in place.
